pkg/cli/token/token.go

```diff
@@ -165,10 +174,30 @@ func Rotate(app *cli.Context) error {
 	}
 	// wait for etcd db propagation delay
 	time.Sleep(1 * time.Second)
-	fmt.Println("token rotated, restart k3s with new token")
+	fmt.Println("Token rotated, restart k3s with new token")
 	return nil
 }
 
+func askContinue(prefix string) bool {
+	reader := bufio.NewReader(os.Stdin)
```

Comment:
Most CLI tools will skip prompting if stdin is not a terminal, is there any existing stuff we could reuse for that here?